airbyte-integrations/connectors/source-mysql/metadata.yaml

```diff
@@ -9,7 +9,7 @@ data:
   connectorSubtype: database
   connectorType: source
   definitionId: 435bb9a5-7887-4809-aa58-28c27df0d7ad
-  dockerImageTag: 3.4.12
+  dockerImageTag: 3.4.13
```

Comment:
request: This should probably be a more obvious change - 3.5.0?

Reply:
Updated here + CDK version as well